src/compiler/emitter.ts

```diff
@@ -1881,13 +1860,77 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
                 }
             }
 
+            function emitCallOrNewExpressionTarget(node: Expression): CallTarget {

```

Comment:
Just tried to address this. The reason I did it was so that I don't need to have multiple node kind checks — one for the getter and one for the emitter. We already have in `emitCallOrNewExpressionTarget()` the node kind checks:
```typescript
if (node.kind === SyntaxKind.PropertyAccessExpression) {
    ...
}
else if (node.kind === SyntaxKind.ElementAccessExpression) {
    ...
}
else if (node.kind === SyntaxKind.SuperKeyword) {
    ...
}
else {
    ...
}
```
If we would have one for getter and emitter the same node kind checks need to be applied mulitple times.

I don't have any perfect solution to this. Any idea how to fix this?